### DISCUSSION TITLE
How to disbale Kitty popup?

### DISCUSSION THREAD

**Discussion by Mosazghi** [kovidgoyal/kitty#8179] [Q&A]
It occurs whenever I'm finished doing something in the terminal. E.g. building a project and kitty will show this:
![image](https://github.com/user-attachments/assets/a1605c65-54eb-4556-b991-1fd900e6d155)

**Comment by SolitudeSF** [comment]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.notify_on_cmd_finish

**Reply by Mosazghi** [reply]
Didnt work

**Comment by kovidgoyal** [comment]
Remove whatever configuration you have done to enable it. Most likely in
your shell rc files.

**Comment by Mosazghi** [comment]
Had to disable it using `dconf-editor` (Ubuntu)

![image](https://github.com/user-attachments/assets/488fa59e-5aa8-4613-be5a-218d607ba5e0)
